src/pages/components/Password/PasswordInput.jsx

```diff
@@ -0,0 +1,28 @@
+import { useState } from "react";
+import { Eye, EyeOff } from "lucide-react";
+import "./PasswordInput.css";
+
+const PasswordInput = ({ showToggle = true }) => {
+  const [showPassword, setShowPassword] = useState(false);
+
+  return (
+    <div className="password-container">
+      <input
+        type={showPassword ? "text" : "password"}
```

Comment:
```suggestion
       type={showPassword ? "password" : "text"}
```
o "olhinho" esta ao contrario, quando ele esta aberto a senha fica invisivel e quando esta fechado a senha fica visivel, mas devia ser o contrario